x-pack/plugins/security_solution/public/detections/components/alerts_table/timeline_actions/use_bulk_add_to_case_actions.tsx

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import { APP_ID } from '../../../../../common/constants';
+import type { TimelineItem } from '../../../../../common/search_strategy';
+import { useKibana } from '../../../../common/lib/kibana';
+import { ADD_TO_CASE_DISABLED, ADD_TO_EXISTING_CASE, ADD_TO_NEW_CASE } from '../translations';
+
+export interface UseAddToCaseActions {
+  onClose?: () => void;
+  onSuccess?: () => Promise<void>;
+}
+
+export const useBulkAddToCaseActions = ({ onClose, onSuccess }: UseAddToCaseActions = {}) => {
```

Comment:
These actions should not be available in the bulk menu if a user does not have 'write' permissions for Cases:

![image](https://user-images.githubusercontent.com/7405507/165940893-eef4cbdf-7e2b-4ed9-aa27-a7b2f0675e79.png)

Currently they are available but lead to an authorization error.

Reply:
Confirmed fixed by https://github.com/elastic/kibana/pull/130958/commits/3a79fd0e1e391c3ae639851e33b3ddcf324a09cb